app/models/miq_report/search.rb

```diff
@@ -40,20 +40,19 @@ def get_cached_page(limit, offset, includes, options)
 
   def get_order_info
     return [] if sortby.nil? # apply limits (note: without order it is non-deterministic)
-    # Convert sort cols from sub-tables from the form of assoc_name.column to the form of table_name.column
-    order = Array.wrap(sortby).collect do |c|
+    # Convert sort cols from sub-tables from the form of assoc_name.column to arel
+    Array.wrap(sortby).collect do |c|
       sql_col, sql_type = association_column(c)
       return nil if sql_col.nil?
-      [:string, :text].include?(sql_type) ? Arel::Nodes::NamedFunction.new('LOWER', [sql_col]) : sql_col
-    end
-
-    if (self.order && self.order.downcase == "descending")
-      order = order.map { |col| Arel::Nodes::Descending.new col }
-    elsif (self.order && self.order.downcase == "ascending")
-      order = order.map { |col| Arel::Nodes::Ascending.new col }
```

Comment:
Thanks for removing local variable order when we also have self.order= and self.order (setter/getter) in play.